spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -241,10 +241,14 @@ function onRecvPacket(packet: Packet) {
   InterchainAccountPacketData data = packet.data
     const tx = deserialiseTx(packet.data.txBytes)
     try {
-      const result = executeTx(tx)
-
+      executeTx(tx)
+      
+      // Gets the interchain account address, and returns it to the controller chain
+      // This step is necessary in order for the controller chain to know the address of the registered interchain account 
+      // We should only return the interchain account on packet sequence 1
+      const interchainAccount = getInterchainAccount(ctx, SourcePortId)
       return Acknowledgement{
-        result: result
+        result: interchainAccount
```

Comment:
Acking since this is a quick change. But this should not just contain the interchain account. Otherwise we don't properly get the result of our first message execution.

We should get the interchain account **in addition** to first message result on ack sequence 1. The way to marshal both in ack, and then unmarshal should be specified in this document.

Will create a separate issue for this to be tackled in subsequent PR